content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,217 @@
 
-In this guide, you do the following:
+## Binary targets in <inlinecode>schema.prisma</inlinecode>
 
-- Use Prisma and a Prisma schema file (`schema.prisma`) to create a schema for the database
-- Use `prisma db push` to push this Prisma schema to the database
+The Prisma schema should contain the following in the `generator` block:
 
-In this guide, you use `prisma db push` to push the initial Prisma schema to the database. For a real project, you would typically start with a development database and use `prisma db push` as we suggest here. You can continue with `prisma migrate dev`, which generates a migration script for each change that you make to the Prisma schema. With multiple migration scripts, you build a migration history for your database and that makes it easy to later apply the changes to a production database with `prisma migrate deploy`. For more information, see [Prisma Migrate](/concepts/components/prisma-migrate). <br></br><br></br>If you use Supabase to host your database, then when you use `prisma migrate` you must set up a shadow database. For details, see the [Supabase docs](https://supabase.com/docs/guides/integrations/prisma).
-
-## 1. Download the example project
-
-To start, download the example project. The project includes the configuration that you need to complete this guide.
-
-### Steps
-
-1. Open a terminal window and navigate to a location of your choice. Use the following commands to create a directory for the project and download the example code:
-
-   ```terminal wrap
-   mkdir prisma-aws-lambda
-   cd prisma-aws-lambda
-   curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=3 prisma-examples-latest/deployment-platforms/aws-lambda/
-   ```
-
-2. Install the dependencies:
-
-   ```terminal
-   npm install
-   ```
-
-<!-- tar strip folder is a concatenation of the REPOSITORY-BRANCH/REF, e.g. prisma-examples-latest -->
-
-### Result
-
-Your directory contains the following files:
-
-```terminal no-copy
-README.md
-handlers
-node_modules
-package-lock.json
-package.json
-prisma
-schema.sql
-serverless.yml
+```prisma
+binaryTargets = ["native", "rhel-openssl-1.0.x"]
 ```
 
-## 2. Set the DATABASE_URL in <inlinecode>.env</inlinecode>
-
-The Lambda functions need the `DATABASE_URL` environment variable to access the database. You must define an `.env` file. The preconfigured `serverless-dotenv-plugin` uses this file to inject into the function runtime.
-
-The repository contains an `.env.example` example file to help with this.
-
-### Steps
+This is necessary because the local runtime is different from the Lambda runtime. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine file available.
```

Comment:
```suggestion
This is necessary because the runtime used in the development and when your application is deployed are different. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine file available.
```